tests/drivers/i2c/i2c_emul/src/emulated_target.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "emulated_target.h"
+
+struct i2c_target_callbacks emulated_callbacks = {
+	.write_requested = NULL,
+	.read_requested = NULL,
+	.write_received = NULL,
+	.read_processed = NULL,
+	.stop = NULL,
+};
+
+struct i2c_target_config emulated_target_config = {
+	.flags = 0,
+	.address = 0x20,
```

Comment:
Can you extract the address from the devicetree? 
What happens if someone updates the devicetree to add a second forwarded target?

Reply:
Done